subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/test/AbstractLaunchRule.kt

```diff
@@ -35,7 +35,7 @@ abstract class AbstractLaunchRule : TestRule {
                 beforeTest()
                 base.evaluate()
             } catch (t: Throwable) {
-                logger.critical("Error during test completion", t)
+                logger.warn("Error during test completion", t)
```

Comment:
We could delete this block because we throw an error further

Reply:
we don't want this test failures in kibana? report viewer is enough?

Reply:
As I understand we will without that log because the test fails further

Reply:
Can you please point me to another place where we log this error?

Reply:
UncaughtReportHandler?

Reply:
If i understand this correctly this exception will be delivered as correct result with `finish()` function, not as an unexpected result

see:
https://github.com/android/android-test/blob/ec8526115902abec5ab0f66af0147874729080f4/runner/android_junit_runner/java/androidx/test/runner/AndroidJUnitRunner.java#L442

and:
https://github.com/android/android-test/blob/ec8526115902abec5ab0f66af0147874729080f4/runner/android_junit_runner/java/androidx/test/internal/runner/TestExecutor.java#L56

Reply:
Ok
```suggestion
                logger.warn("Test execution failed", t)
```